developer-docs/upgrade/build_n_deploy.md

```diff
@@ -22,57 +22,45 @@ As part of this upgrade, you may choose to set up the load balancer for swarm ma
+| SyncTool                | Neo4jElasticSearchSyncTool(Please see below notes to choose jenkins job parameter)	                  |                    | KafkaSetup          | Keycloak  | Keycloak  |                         
 |                         |                           |                    |                     |           | ApplicationElasticSearch
-|                         | StartNeo4jCluster	      |                    | KafkaIndexer        | Player    | OpsAdministration/Core/ESMapping (Provide value as `all` for job parameter indices_name)    |
-|                         | KafkaSetup                | Secor              | Secor               | Learner   | Player   |
-|                         | CassandraDbUpdate         |	Analytics          | AnalyticsAPI        | Content   | Learner   |
-|                         | CassandraTrigger          |                    |                     | Lms       | Content       |
-| CassandraTrigger        | Neo4jDefinitionUpdate     | DataPipeline       |	DataProducts       | Telemetry | Lms |
-| KnowledgePlatform       |	Learning                  |                    | SamzaTelemetrySchemas | Proxy   | Telemetry     |
-|                         | Search	                  | Yarn               |	Yarn (Multiselect all options in the job parameter job_names_to_deploy)	             |           | OnboardAPI|
-|  Yarn	                  | Yarn                      |                    |                     |           | Proxy |
-|  SyncTool               | Neo4jElasticSearchSyncTool|                    |                     |           | OnboardConsumers   |
+|                         | Neo4jDefinitionUpdate 	      |                    | KafkaIndexer        | Player    | OpsAdministration/Core/ESMapping (Provide value as `all` for job parameter indices_name)    |
+|                         |                 | Secor              | Secor               | Learner   | Player   |
+|                         |          |	Analytics          | AnalyticsAPI        | Content   | Learner   |
+|                         |              |                    | Provision/AnalyticsSpark  | Lms       | Content       |
+|                         |       | DataPipeline       |	DataProducts       |  | Lms |
+|                         | 	                  | Yarn               |	Yarn (Multiselect all options in the job parameter job_names_to_deploy)	             | EncService           | EncService |
+|  	                  |                       |                    | AnalyticsGeoLocationDBSetup | Cert           | Cert |
 
 
-**Note:** 
-1. We have disabled Git LFS for private repositories. Hence, uninstall git lfs from your private repository.
-To do so, clone your private repository and run ```git lfs uninstall``` and also remove the ```.gitattributes``` file
+> **Note:** 
+1. Refer to the following notes to trigger the **Neo4jElasticSearchSyncTool** job.
+**Steps to migrate DialcodeRequired:**
```

Comment:
@Kaali09 could you please check if the rephrasing of this sentence works? If not, please tell me what exactly do you mean, then we can phrase it differently.

Reply:
@Sunitashelar I have Worked on your comment. Please verify.

Reply:
done